packages/perl.rb

```diff
@@ -7,19 +7,6 @@ class Perl < Package
   source_url 'http://www.cpan.org/src/5.0/perl-5.26.1.tar.gz'
   source_sha256 'e763aa485e8dc1a70483dbe6d615986bbf32b977f38016480d68c99237e701dd'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/perl-5.26.1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/perl-5.26.1-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/perl-5.26.1-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/perl-5.26.1-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: 'd905b33e42f8825063eebe17a34bc8011ed9e7a88374373187d55f879453ccc4',
-     armv7l: 'd905b33e42f8825063eebe17a34bc8011ed9e7a88374373187d55f879453ccc4',
-       i686: 'a1394c4caafbca33de118521f850fa4162834f70c4bacbca9b576048a420ffb2',
-     x86_64: 'd6f71ac89fae67ce77d097075b52683cc5bb6367f87ad2dc86c383b4024b4186',
-  })
-
   depends_on 'patch' => :build
```

Comment:
What is changing here?  I only noticed the binaries were removed with no other changes?